core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -91,6 +91,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
     logError(processErr.toString)
 
 
+    if(!new File(hc.getConf.clusterInfoFile.get).exists()){
```

Comment:
One small nitpick.
It makes sense to stick to the coding style that IntelliJ supports as default. Meaning, 
if (!new File(hc.getConf.clusterInfoFile.get).exists()) {
Of course, since we are here, could also check that the file is readable, and that hc.getConf.clusterInfoFile.get does not throw an NPE and does not return a null. :) (Just commenting)